MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -843,6 +844,29 @@ public void DrawIndexedPrimitives(PrimitiveType primitiveType, int baseVertex, i
             PlatformDrawIndexedPrimitives(primitiveType, baseVertex, startIndex, primitiveCount);
         }
 
+        public void DrawIndexedPrimitives(PrimitiveType primitiveType, int baseVertex, int startIndex, int primitiveCount)
+        {
+            if (_vertexShader == null)
+                throw new InvalidOperationException("Vertex shader must be set before calling DrawIndexedPrimitives.");
+
+            if (_vertexBuffer == null)
+                throw new InvalidOperationException("Vertex buffer must be set before calling DrawIndexedPrimitives.");
+
+            if (_indexBuffer == null)
+                throw new InvalidOperationException("Index buffer must be set before calling DrawIndexedPrimitives.");
+
+            if (primitiveCount <= 0)
+                throw new ArgumentOutOfRangeException("primitiveCount");
+
+            unchecked
+            {
+                _graphicsMetrics._drawCount++;
+                _graphicsMetrics._primitiveCount += (ulong)primitiveCount;
+            }
+
+            PlatformDrawIndexedPrimitives(primitiveType, baseVertex, startIndex, primitiveCount);
+        }

```

Comment:
> I also wouldn't change the parameter name.

I guess I suggested to change the parameter name so that no one gets sucked into trying to figure out what their `minVertexIndex` is before finally noticing it is unused.

I'm fine doing it either way.